templates/registration/complete.html

```diff
@@ -11,6 +11,10 @@
     <script type='text/javascript' src="{%static "jquery-file-download/src/Scripts/jquery.fileDownload.js"%}"></script>
   </head>
   <body>
+    <p>
+      You're about to download a .json file uniquely generated for your account. This file stores you api key.<br>
+      Give it a name and store it in a place you'll remember. <b>You'll need it every time you login</b>, in addition to you username and password.
```

Comment:
Do a Cmd + F on "in addition to you username and password." and you'll find it :)